lib/auth-client.js

```diff
@@ -146,11 +148,14 @@ AuthClient.prototype.getAuthUrl = function(options) {
   }
 
   let mode;
-  if (this.development !== undefined) {
+  if (options.testMode !== undefined) {
+    mode = options.testMode;
```

Comment:
could add a comment to explain this is still supporting `development`

Reply:
Just added!